plugins/services/src/js/components/SDKPlans.tsx

```diff
@@ -127,7 +127,11 @@ const SDKPlans = componentFromStream(props$ => {
         concat(throwError(errors))
       )
     ),
-    catchError(() => of(<ErrorScreen />)),
+    catchError(e => {
+      console.warn(e);
```

Comment:
Build is failing because of this line.

Reply:
removed the according commit for now to come up with proper logging-facilities.